browser/js/views/lecture/lectureModal.html

```diff
@@ -3,7 +3,17 @@
     <h1>Create Lecture</h1>
   </div>
   <div class="modal-body">
-    <input class="mainQuestion form-control" type="text" ng-model='lectureName' ng-keyup="$event.keyCode == 13 ? submitLecture() : null" placeholder='Lecture Title'>
+    <input class="mainQuestion form-control" type="text" ng-model='lectureName' placeholder='Lecture Title'>
+    <select id='lectureEmail' ng-model='lectureEmail'>
+      <option value="" selected>Please select an instructor</option>
+      <option value="gabriel@fullstackacademy.com">Gabe Lebec</option>
```

Comment:
Why not `ng-repeat`—or better yet [`ng-options`](https://docs.angularjs.org/api/ng/directive/ngOptions)?